packages/database/src/prisma-api.ts

```diff
@@ -520,7 +536,9 @@ export function getSdk(
         data: {
           sourceUserId: userId,
           purchaseId: purchaseId,
-          expiresAt: new Date(Date.now() + oneWeekInMilliseconds),
+          expiresAt: existingPurchase
+            ? new Date()
```

Comment:
goofy, but I didn't know how this would affect the transaction so it just expires in place and will eventually get cleaned up

we don't want transfers on upgrades